tools/styles/tokens/global-vars.css

```diff
@@ -129,6 +129,19 @@ governing permissions and limitations under the License.
     --spectrum-floating-action-button-shadow-color: var(
         --spectrum-floating-action-button-drop-shadow-color
     );
+    --spectrum-table-row-hover-color: var(--spectrum-gray-900);
+    --spectrum-table-row-hover-opacity: 0.07;
+    --spectrum-table-selected-row-background-color: var(
+        --spectrum-informative-background-color-default
+    );
+    --spectrum-table-selected-row-background-opacity: 0.1;
+    --spectrum-table-selected-row-background-color-non-emphasized: var(
+        --spectrum-neutral-background-color-selected-default
+    );
+    --spectrum-table-selected-row-background-opacity-non-emphasized: 0.1;
+    --spectrum-table-row-down-opacity: 0.1;
```

Comment:
This instead?
 --spectrum-table-row-down-opacity: var(--spectrum-table-selected-row-background-opacity)

Reply:
This would be a useful question for the Spectrum CSS team, as we do not write this file. It is mechanically adopted from their `@spectrum-css/tokens` file.